pkg/broker/ingress/handler.go

```diff
@@ -24,18 +24,22 @@ import (
 	"strings"
 	"time"
 
-	_ "knative.dev/pkg/metrics/testing"
-
 	cev2 "github.com/cloudevents/sdk-go/v2"
 	"github.com/cloudevents/sdk-go/v2/binding"
 	"github.com/cloudevents/sdk-go/v2/binding/transformer"
 	"github.com/cloudevents/sdk-go/v2/protocol"
 	"github.com/cloudevents/sdk-go/v2/protocol/http"
 	"github.com/google/knative-gcp/pkg/metrics"
+	"github.com/google/knative-gcp/pkg/tracing"
 	"github.com/google/wire"
+	"go.opencensus.io/trace"
 	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/types"
 	"knative.dev/eventing/pkg/kncloudevents"
 	"knative.dev/eventing/pkg/logging"
+	kntracing "knative.dev/eventing/pkg/tracing"
+
+	_ "knative.dev/pkg/metrics/testing"
```

Comment:
Why is a testing package being used in production code?

I realize you just moved this line, but can it be removed instead?